modules/apps/image/image-impl/src/main/java/com/liferay/image/internal/ImageMagickImpl.java

```diff
@@ -226,7 +227,11 @@ public byte[] scale(byte[] bytes, String mimeType, int width, int height)
 
 			Future<?> future = convert(arguments);
 
-			future.get();
+			ProcessEvent processEvent = (ProcessEvent)future.get();
+
+			if (_log.isDebugEnabled() && processEvent.getException() != null) {
```

Comment:
There's an SF issue here; the second subexpression should be bracketed `... && (processEvent.getException() ...))`

Reply:
Thanks @adolfopa, sorry about that. I've updated it now.